.github/workflows/e2e-test-with-domain.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
         working-directory: test/
         run: |
           set -o pipefail
-          go test ./e2etest -v -timeout 85m -run Installer | tee ./e2etest/artifacts/e2etest.log
+          go test ./e2etest -v -timeout 95m -run Installer | tee ./e2etest/artifacts/e2etest.log
```

Comment:
Im sure this is a value we've changed a few times in the past before? What might be nice is to turn this into a github variable which can then be changed dynamically as needs be eliminating future code changes.

Reply:
Hey @errcode1202 , sorry about the confusion .. The PR is still in draft, and I made these change only for testing purpose. It'll be reverted back. Same as the other comment in install.sh.

Reply:
👍🏼